routes/api/users.js

```diff
@@ -45,7 +48,9 @@ router.post ('/signup', (req, res) => {
             .save ()
             .then (user => {
               const payload = {id: user.id, username: user.username};
-
+              const calendar = new Calendar({user: user.id});
+              calendar.save().then(console.log('sucess'));
```

Comment:
Can we either remove the `then` entirely or confirm with something other than the console?